src/common/navigation.yaml

```diff
@@ -77,24 +77,7 @@ sections:
           - path: /install
           - path: /usage
           - path: /security
-      - path: /naming-services
-        pages:
-          - path: /overview
-          - path: /architecture
-          - path: /comparison
-          - path: /did
-          - path: /forks
-          - path: /namespaces
-          - path: /subdomains
-        sections:
-          - title: Tutorials
-            pages:
-              - path: /choose-name
-              - path: /create-namespace
-              - path: /resolve-name
-              - path: /register-name
-              - path: /manage-names
-              - path: /subdomains-tutorial
+      - path: /naming-system # is an overview page
```

Comment:
These need to be redirected to `/naming services`, see here https://github.com/blockstack/docs/blob/master/next.config.js#L9

Reply:
I see - so I should add redirects to the `next.config.js` that redirect `/naming-services/*` to `/naming-system` ?

Reply:
(And fix any existing redirects)

Reply:
Updated in ca79fea1